### DISCUSSION TITLE
Is there a communication platform where people can discuss simulately?

### DISCUSSION THREAD

**Discussion by Daoming-Chen** [geng-haoran/Simulately#25] [Ideas]
Great work! I find this webset very useful.

I wonder do you have a online chat group?

I think it would a good idea to have a discord or wechat group for people to talk to each other.

**Comment by geng-haoran** [comment]
Great, please join us through slack [here](https://join.slack.com/t/simulately/shared_invite/zt-2jtkhq8k7-AErgkbD8Y3iBy1CqgtlIHg)
